layouts/partials/footer.html

```diff
@@ -67,12 +67,12 @@
            <span class="footer-head">Social</span>
                <ul class="footer-social">
                  <li><a href="https://www.facebook.com/nginxinc" title="facebook" aria-label="facebook" target="_blank" rel="noreferrer"><i class="fa-brands fa-facebook" aria-hidden="true"></i>
-                   Facebook</a></li>
-                 <li><a href="https://twitter.com/nginx" title="twitter" aria-label="twitter" target="_blank" rel="noreferrer"><i class="fa-brands fa-twitter" aria-hidden="true"></i>Twitter</a></li>
+                  </a></li>
+                 <li><a href="https://twitter.com/nginx" title="X" aria-label="x" target="_blank" rel="noreferrer"><i class="fa-brands fa-x-twitter" aria-hidden="true"></i></a></li>
```

Comment:
```suggestion
                 <li><a href="https://twitter.com/nginx" title="X" aria-label="X" target="_blank" rel="noreferrer"><i class="fa-brands fa-x-twitter" aria-hidden="true"></i></a></li>
```

LOGAF because social media but the capital X is the actual name: lowercase looks like it could be a placeholder.